deploy/kubernetes/dev/controller.yaml

```diff
@@ -42,7 +42,7 @@ spec:
               mountPath: /csi
         - name: gce-pd-driver
           imagePullPolicy: Always
-          image: gcr.io/google-containers/volume-csi/gcp-compute-persistent-disk-csi-driver:v0.1.0.alpha
+          image: gcr.io/dyzz-csi-staging/csi/gce-pd-driver:latest
```

Comment:
You should look into kustomize at a later time.  It lets you patch specs with only the difference.  So anytime you want to change your specs, you don't have to change it both places.